cdap-watchdog/src/main/java/co/cask/cdap/logging/write/FileMetaDataManager.java

```diff
@@ -71,21 +89,35 @@ public FileMetaDataManager(final LogSaverTableUtil tableUtil, TransactionExecuto
   }
 
   /**
-   * Persistes meta data associated with a log file.
+   * Persists meta data associated with a log file.
+   *
    * @param loggingContext logging context containing the meta data.
-   * @param startTimeMs start log time associated with the file.
-   * @param location log file.
+   * @param startTimeMs    start log time associated with the file.
```

Comment:
no extra spaces. Our convention for javadocs is to just have a single space and not to align everything